modules/nrf_wifi/bus/Kconfig

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+DT_COMPAT_NORDIC_NRF70_QSPI := nordic,nrf7002-qspi
```

Comment:
This won't work with nRF7000/nRF7001, need to add entries for both.

Reply:
Added both now